[LBSE] Add remaining compositing tests from LBSE downstream

nikolaszimmermann opened the pull request:
#### 69d1edc2e19666330bb9511b2d2d60725c309c6c
<pre>
[LBSE] Add remaining compositing tests from LBSE downstream
<a href="https://bugs.webkit.org/show_bug.cgi?id=247667">https://bugs.webkit.org/show_bug.cgi?id=247667</a>

Reviewed by Rob Buis.

Upstream remaining compositing tests - most of them are basic and helped
brining up compositing in LBSE. All tests pass in LBSE already.

No change in functionality, only adding new tests.

* LayoutTests/svg/compositing/container-after-composited-child-expected.svg: Added.
* LayoutTests/svg/compositing/container-after-composited-child.svg: Added.
* LayoutTests/svg/compositing/container-before-composited-child-expected.svg: Added.
* LayoutTests/svg/compositing/container-before-composited-child.svg: Added.
* LayoutTests/svg/compositing/outermost-svg-directly-composited-group-child-overflow-hidden.html:
* LayoutTests/svg/compositing/rect-as-container-child-1-expected.svg: Added.
* LayoutTests/svg/compositing/rect-as-container-child-1.svg: Added.
* LayoutTests/svg/compositing/rect-as-container-child-2-expected.svg: Added.
* LayoutTests/svg/compositing/rect-as-container-child-2.svg: Added.
* LayoutTests/svg/compositing/rect-as-container-child-3-expected.svg: Added.
* LayoutTests/svg/compositing/rect-as-container-child-3.svg: Added.
* LayoutTests/svg/compositing/rect-as-transformed-container-child-expected.svg: Added.
* LayoutTests/svg/compositing/rect-as-transformed-container-child.svg: Added.
* LayoutTests/svg/compositing/rect-direct-svg-root-child-expected.svg: Added.
* LayoutTests/svg/compositing/rect-direct-svg-root-child.svg: Added.
* LayoutTests/svg/compositing/rect-in-nested-transformed-container-1-expected.svg: Added.
* LayoutTests/svg/compositing/rect-in-nested-transformed-container-1.svg: Added.
* LayoutTests/svg/compositing/rect-in-nested-transformed-container-2-expected.svg: Added.
* LayoutTests/svg/compositing/rect-in-nested-transformed-container-2.svg: Added.
* LayoutTests/svg/compositing/rect-in-nested-transformed-container-3-expected.svg: Added.
* LayoutTests/svg/compositing/rect-in-nested-transformed-container-3.svg: Added.
* LayoutTests/svg/compositing/rect-in-nested-transformed-container-4-expected.svg: Added.
* LayoutTests/svg/compositing/rect-in-nested-transformed-container-4.svg: Added.
* LayoutTests/svg/compositing/rect-in-nested-transformed-container-5-expected.svg: Added.
* LayoutTests/svg/compositing/rect-in-nested-transformed-container-5.svg: Added.
* LayoutTests/svg/compositing/rect-in-nested-transformed-container-6-expected.svg: Added.
* LayoutTests/svg/compositing/rect-in-nested-transformed-container-6.svg: Added.
* LayoutTests/svg/compositing/transformed-container-after-composited-child-expected.svg: Added.
* LayoutTests/svg/compositing/transformed-container-after-composited-child.svg: Added.
* LayoutTests/svg/compositing/transformed-container-before-composited-child-expected.svg: Added.
* LayoutTests/svg/compositing/transformed-container-before-composited-child.svg: Added.
* LayoutTests/svg/compositing/transformed-rect-as-container-child-expected.svg: Added.
* LayoutTests/svg/compositing/transformed-rect-as-container-child.svg: Added.
* LayoutTests/svg/compositing/transformed-rect-as-transformed-container-child-1-expected.svg: Added.
* LayoutTests/svg/compositing/transformed-rect-as-transformed-container-child-1.svg: Added.
* LayoutTests/svg/compositing/transformed-rect-as-transformed-container-child-2-expected.svg: Added.
* LayoutTests/svg/compositing/transformed-rect-as-transformed-container-child-2.svg: Added.
* LayoutTests/svg/compositing/transformed-rect-as-transformed-container-child-3-expected.svg: Added.
* LayoutTests/svg/compositing/transformed-rect-as-transformed-container-child-3.svg: Added.
* LayoutTests/svg/compositing/transformed-rect-direct-svg-root-child-expected.svg: Added.
* LayoutTests/svg/compositing/transformed-rect-direct-svg-root-child.svg: Added.

Canonical link: <a href="https://commits.webkit.org/256502@main">https://commits.webkit.org/256502@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/1a70155f3ed5c01ee7d1e19ecc07bec19839409d

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/95921 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/5170 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/28964 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/105481 "Built successfully") | [✅ 🛠 🧪 win](https://ews-build.webkit.org/#/builders/10/builds/165799 "Built successfully and passed tests") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/5271 "Built successfully") | [✅ 🛠 mac-debug](https://ews-build.webkit.org/#/builders/71/builds/33931 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/88299 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/101309 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/101583 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/3881 "Passed tests") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/82528 "Built successfully") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/30920 "Passed tests") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/9/builds/85735 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/87643 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/73753 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/66/builds/39662 "Built successfully") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/73/builds/19176 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/67/builds/37342 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/20512 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/4495 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/63/builds/41814 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/60/builds/43131 "Passed tests") | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/65/builds/43720 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/39768 "Passed tests") | | 
<!--EWS-Status-Bubble-End-->

rwlbuis reviewed (approved):
LGTM (if all turns out green).

nikolaszimmermann committed:
[LBSE] Add remaining compositing tests from LBSE downstream
https://bugs.webkit.org/show_bug.cgi?id=247667

Reviewed by Rob Buis.

Upstream remaining compositing tests - most of them are basic and helped
brining up compositing in LBSE. All tests pass in LBSE already.

No change in functionality, only adding new tests.

* LayoutTests/svg/compositing/container-after-composited-child-expected.svg: Added.
* LayoutTests/svg/compositing/container-after-composited-child.svg: Added.
* LayoutTests/svg/compositing/container-before-composited-child-expected.svg: Added.
* LayoutTests/svg/compositing/container-before-composited-child.svg: Added.
* LayoutTests/svg/compositing/outermost-svg-directly-composited-group-child-overflow-hidden.html:
* LayoutTests/svg/compositing/rect-as-container-child-1-expected.svg: Added.
* LayoutTests/svg/compositing/rect-as-container-child-1.svg: Added.
* LayoutTests/svg/compositing/rect-as-container-child-2-expected.svg: Added.
* LayoutTests/svg/compositing/rect-as-container-child-2.svg: Added.
* LayoutTests/svg/compositing/rect-as-container-child-3-expected.svg: Added.
* LayoutTests/svg/compositing/rect-as-container-child-3.svg: Added.
* LayoutTests/svg/compositing/rect-as-transformed-container-child-expected.svg: Added.
* LayoutTests/svg/compositing/rect-as-transformed-container-child.svg: Added.
* LayoutTests/svg/compositing/rect-direct-svg-root-child-expected.svg: Added.
* LayoutTests/svg/compositing/rect-direct-svg-root-child.svg: Added.
* LayoutTests/svg/compositing/rect-in-nested-transformed-container-1-expected.svg: Added.
* LayoutTests/svg/compositing/rect-in-nested-transformed-container-1.svg: Added.
* LayoutTests/svg/compositing/rect-in-nested-transformed-container-2-expected.svg: Added.
* LayoutTests/svg/compositing/rect-in-nested-transformed-container-2.svg: Added.
* LayoutTests/svg/compositing/rect-in-nested-transformed-container-3-expected.svg: Added.
* LayoutTests/svg/compositing/rect-in-nested-transformed-container-3.svg: Added.
* LayoutTests/svg/compositing/rect-in-nested-transformed-container-4-expected.svg: Added.
* LayoutTests/svg/compositing/rect-in-nested-transformed-container-4.svg: Added.
* LayoutTests/svg/compositing/rect-in-nested-transformed-container-5-expected.svg: Added.
* LayoutTests/svg/compositing/rect-in-nested-transformed-container-5.svg: Added.
* LayoutTests/svg/compositing/rect-in-nested-transformed-container-6-expected.svg: Added.
* LayoutTests/svg/compositing/rect-in-nested-transformed-container-6.svg: Added.
* LayoutTests/svg/compositing/transformed-container-after-composited-child-expected.svg: Added.
* LayoutTests/svg/compositing/transformed-container-after-composited-child.svg: Added.
* LayoutTests/svg/compositing/transformed-container-before-composited-child-expected.svg: Added.
* LayoutTests/svg/compositing/transformed-container-before-composited-child.svg: Added.
* LayoutTests/svg/compositing/transformed-rect-as-container-child-expected.svg: Added.
* LayoutTests/svg/compositing/transformed-rect-as-container-child.svg: Added.
* LayoutTests/svg/compositing/transformed-rect-as-transformed-container-child-1-expected.svg: Added.
* LayoutTests/svg/compositing/transformed-rect-as-transformed-container-child-1.svg: Added.
* LayoutTests/svg/compositing/transformed-rect-as-transformed-container-child-2-expected.svg: Added.
* LayoutTests/svg/compositing/transformed-rect-as-transformed-container-child-2.svg: Added.
* LayoutTests/svg/compositing/transformed-rect-as-transformed-container-child-3-expected.svg: Added.
* LayoutTests/svg/compositing/transformed-rect-as-transformed-container-child-3.svg: Added.
* LayoutTests/svg/compositing/transformed-rect-direct-svg-root-child-expected.svg: Added.
* LayoutTests/svg/compositing/transformed-rect-direct-svg-root-child.svg: Added.

Canonical link: https://commits.webkit.org/256502@main
